server/src/test/java/org/spine3/server/command/CommandSchedulingShould.java

```diff
@@ -121,7 +121,7 @@ public void reschedule_commands_from_storage_in_parallel_on_build_if_thread_spaw
 
         // Sleep to ensure the commands have been rescheduled in parallel.
         try {
-            Thread.sleep(100);
+            Thread.sleep(150);
```

Comment:
Because that test failed yesterday during the check after updating the branch.
https://travis-ci.org/SpineEventEngine/core-java/builds/204632979
Also one more build was failed yesterday, I think it is because our tests are super fast.
https://travis-ci.org/SpineEventEngine/core-java/builds/204663484